Dockerfile

```diff
@@ -30,18 +37,20 @@ ENV PROJ_LIB=/opt/conda/share/proj
 # When using pip in a conda environment, conda commands should be ran first and then
 # the remaining pip commands: https://www.anaconda.com/using-pip-in-a-conda-environment/
 RUN conda install -c conda-forge matplotlib basemap cartopy python-igraph imagemagick pysal && \
-    conda install -c h2oai h2o && \
-    conda install -c pytorch pytorch torchvision torchaudio cpuonly && \
+    # b/142337634#comment22 pin required to avoid torchaudio downgrade.
+    conda install -c pytorch pytorch torchvision "torchaudio>=0.4.0" cpuonly && \
     /tmp/clean-layer.sh
 
 # The anaconda base image includes outdated versions of these packages. Update them to include the latest version.
 # b/150498764 distributed 2.11.0 fails at import while trying to reach out to 8.8.8.8 since the network is disabled in our hermetic tests.
 RUN pip install distributed==2.10.0 && \
-    # b/145358669 remove --upgrade once we upgrade base image which will include numpy >= 1.17
-    pip install --upgrade numpy && \
     pip install seaborn python-dateutil dask && \
     pip install pyyaml joblib pytagcloud husl geopy ml_metrics mne pyshp && \
     pip install pandas && \
+    # Install h2o from source.
+    # Use `conda install -c h2oai h2o` once Python 3.7 version is released to conda.
+    apt-get install -y default-jre && \
```

Comment:
Do you know if h2o uses java to draw anything? Otherwise jre-headless could be lighterweight.

Reply:
I filed https://b.corp.google.com/issues/153753638 to look into it.